lib/rounds.js

```diff
@@ -3,8 +3,18 @@ class Round {
     this.categories = [];
     this.clues = []; 
     this.createCategories();
+    this.dailyDouble = null;
+    this.assignDailyDouble();
```

Comment:
I wonder if we can invoke these methods within a different method/file instead of within the constructor.